src/common/utilities/EditableFileSystemProvider.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import * as vscode from 'vscode';
+
+
+export class EditableFileSystemProvider implements vscode.FileSystemProvider {
+    private fileContentMap: { [key: string]: Uint8Array } = {};
```

Comment:
Updated, earlier I add that _ since second property was an even emitter but now, I think since both are private members same convention should be used.